Language/Structure/Further Syntax/define.adoc

```diff
@@ -24,7 +24,7 @@ subCategories: [ "Further Syntax" ]
 This can have some unwanted side effects though, if for example, a constant name that had been #defined is included in some other constant or variable name. In that case the text would be replaced by the #defined number (or text).
 [%hardbreaks]
 
-In general, the `link:../../../variables/variable-scope\--qualifiers/const[const]` keyword is preferred for defining constants and should be used instead of `#define`.
+In general, the `link:../../variables/variable-scope-qualifiers/const[const]` keyword is preferred for defining constants and should be used instead of `#define`.
```

Comment:
Regarding the extra `-` that is breaking the links to the pages under the "Variable Scope & Qualifiers" section, I have verified that this worked correctly up until recently, as you can see from the Wayback Machine's archive of the page:
http://web.archive.org/web/20190916201140/https://www.arduino.cc/reference/en/language/variables/variable-scope--qualifiers/const/

I've notified Arduino's developers of this change, so I'll wait to see whether the dash needs to be removed from the URLs before taking any action on this issue.